falcon/response.py

```diff
@@ -1014,6 +1015,24 @@ def append_link(
         format_content_disposition,
```

Comment:
I think it would make sense to use `functools.partial(format_content_disposition, disposition_type='attachment'),`
so that's cleared to the reader, what's passed

Reply:
Done